src/nvim/os/fs.c

```diff
@@ -192,13 +192,15 @@ int os_open(const char* path, int flags, int mode)
 
 /// Get stat information for a file.
 ///
-/// @return OK on success, FAIL if a failure occurred.
+/// @return `true` on success, `false otherwise.
 static bool os_stat(const char *name, uv_stat_t *statbuf)
   FUNC_ATTR_NONNULL_ALL
 {
   uv_fs_t request;
```

Comment:
This should make the error go away, although I wonder if it is covering up a deeper bug (perhaps in libuv?):

    uv_fs_t request = {0}